reference/checkouts.sf.yml

```diff
@@ -2626,743 +1363,8 @@ components:
-                                type: object
-                                properties:
-                                  name:
-                                    type: string
-                                    description: The name provided by the merchant.
-                                  discountedAmount:
-                                    type: number
-                                    description: The discounted amount applied within a given context.
-                                    format: double
-                            discountAmount:
-                              type: number
-                              description: The total value of all discounts applied to this item (excluding coupon).
-                              format: double
-                            couponAmount:
-                              type: number
-                              description: The total value of all coupons applied to this item.
-                              format: double
-                            originalPrice:
-                              type: number
-                              description: The item’s original price is the same as the product’s default price.
-                            listPrice:
-                              type: number
-                              description: The item’s list price, as quoted by the manufacturer or distributor.
-                              format: double
-                            salePrice:
-                              type: number
-                              description: The itemʼs price after all discounts are applied. (The final price before tax calculation.)
-                              format: double
-                            extendedListPrice:
-                              type: number
-                              description: The itemʼs list price multiplied by the quantity.
-                              format: double
-                            extendedSalePrice:
-                              type: number
-                              description: The itemʼs sale price multiplied by the quantity.
-                              format: double
-                            type:
-                              type: string
-                              description: the product type - physical or digital
-                            isMutable:
-                              type: boolean
-                              description: ''
-                            isShippingRequired:
-                              type: boolean
-                              description: Whether this item requires shipping to a physical address.
-                            downloadFileUrls:
-                              type: array
-                              description: URLs to download all product files.
-                              items:
-                                type: string
-                            downloadPageUrl:
```

Comment:
Is this field deprecated? I can mark it as deprecated instead of removing it

Reply:
Please remove this field, it should only be available once an order has been created. 🙇